src/microbe_stage/AgentProjectile.cs

```diff
@@ -18,6 +18,33 @@ public class AgentProjectile : RigidBody, ITimedLife
     [JsonProperty]
     private float? FadeTimeRemaining { get; set; }
 
+    public static AgentProjectile SpawnAgent(AgentProperties properties, float amount,
```

Comment:
If this is now included in this class this could be named just `Spawn`
this same comment probably applies to the other spawn methods as well.